docs/install/lin/activating-a-validator.md

```diff
@@ -48,22 +53,27 @@ This command will output a `Raw Transaction Data` block:
 
 **Method 2:** Users may also choose to submit the required 32 GöETH along with the data directly to the current deposit contract ****\(found [here](https://prylabs.net/contract)\). To enable the hex data field on the Send page in Metamask, click your wallets avatar &gt; Settings &gt; Advanced &gt; toggle 'Show hex data'.
 
-## Starting up the beacon node
+## Step 4a: Starting up the beacon node
+
+**NOTICE:** If you have already started and syncronised your beacon node, this portion can be skipped.
```

Comment:
These here should have a ` >` character in front of them to make it into a tip box, I think there's a couple missing on the validator pages actually. but not critical and can be added later

Reply:
fixed